src/features/Helpers/configuration.ts

```diff
@@ -69,32 +69,26 @@ export class Configuration {
       .get("enabled");
   }
 
-  public static executeInTerminal(): boolean {
-    return vscode.workspace
-      .getConfiguration("sqlfluff.experimental.format")
-      .get("executeInTerminal");
-  }
-
   public static runTrigger(): string {
     return vscode.workspace
       .getConfiguration("sqlfluff.linter")
       .get("run");
   }
 
   public static lintBufferArguments(): string[] {
-    return ["lint", "--format", "json", "-"];
+    return ["--format", "json"];
   }
```

Comment:
This can be removed and just use `lintFileArguments` instead as they now return the same values.

Reply:
so, i was conflicted about this. the issue in my mind is that some people _may_ have modified these already, and removing one might cause unintended side effects. i'm definitely more interested in cutting them down, so if thats the way you'd like to go happy to do so.

Reply:
I don't quite follow? While these are in the **configuration.ts** file, they can't actually be modified by the user. These arguments should stay consistent. And it looks like you moved the `"-"` part of it to the section where you are evaluating if the linting should be completed with stdin or not. So we should be okay removing this and any places that use it could just use the `lintFileArguments`